README.md

```diff
@@ -7,13 +7,13 @@
 
 The SmartcarAuth iOS SDK makes it easy to integrate with Smartcar Connect from iOS.
 
-The SDK follows the best practices set out in [OAuth 2.0 for Native Apps](https://tools.ietf.org/html/draft-ietf-oauth-native-apps-06) including using _SFSafariViewController_, _SFAuthenticationSession_, or _ASWebAuthenticationSession_ depending on iOS version for the authorization request. _UIWebView_ is explicitly not supported due to usability and security reasons.
+The SDK follows the best practices set out in [OAuth 2.0 for Native Apps](https://tools.ietf.org/html/draft-ietf-oauth-native-apps-06).
 
 ## Requirements
 
-SmartcarAuth supports iOS 10 and above.
+SmartcarAuth supports iOS 13 and above.
```

Comment:
Yes and no, if we wanted to support iOS 12 and below we could probably use another WebView instead of `ASWebAuthenticationSession`, of which some of the supporting classes I believe require iOS 13 and above. Since the [oldest iOS version supported by Apple is iOS 15](https://en.wikipedia.org/wiki/IOS_version_history#Overview), I am comfortable dropping support for iOS 12 and under.